### PR TITLE
Bunch of improvements

### PR DESCRIPTION
**Pull request by dgusakov**
chore: Use dynamic limit for total unstaked alerts

chore: Double fetch storage slot in case of null value
  
chore: Make pool size increase alerts info
  
chore: Add HIGH severity to rewards, decrease alert

feat: ENS expiration alerts

### CODE REVIEW

File: ethereum-financial/src/agent-pools-balances.ts [hunk 1]
```diff
@@ -330,10 +340,13 @@ async function handleCurvePoolSize(
...
   const poolSizeChange = calcChange(poolParams.poolSize, poolSize);
   if (Math.abs(poolSizeChange) > POOL_SIZE_CHANGE_TOLERANCE_INFO) {
-    const severity =
+    let severity =
       Math.abs(poolSizeChange) > POOL_SIZE_CHANGE_TOLERANCE_HIGH
         ? FindingSeverity.High
         : FindingSeverity.Info;
+    if (poolSizeChange > 0) {
+      severity = FindingSeverity.Info;
+    }
```
Comment (vgorkavenko) [inline ethereum-financial/src/agent-pools-balances.ts:349]:
```suggestion
    let severity = FindingSeverity.Info;
    if (Math.abs(poolSizeChange) > POOL_SIZE_CHANGE_TOLERANCE_HIGH) {
      severity = FindingSeverity.High
    }
```

File: ethereum-operations/src/agent-dao-ops.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import { ethersProvider, etherscanProvider } from "./ethers";
 import NODE_OPERATORS_REGISTRY_ABI from "./abi/NodeOperatorsRegistry.json";
 import LIDO_DAO_ABI from "./abi/LidoDAO.json";
 import MEV_ALLOW_LIST_ABI from "./abi/MEVBoostRelayAllowedList.json";
+import ENS_BASE_REGISTRAR_ABI from "./abi/ENS.json";
```
Comment (vgorkavenko) [inline ethereum-operations/src/agent-dao-ops.ts:17]: Where is this file?

File: ethereum-operations/src/agent-dao-ops.ts [hunk 3]
```diff
@@ -415,6 +422,47 @@ async function handleMevRelayCount(
...
+    );
+    await Promise.all(
+      LIDO_ENS_NAMES.map(async (name) => {
+        const labelHash = ethers.utils.keccak256(
+          ethers.utils.toUtf8Bytes(name)
+        );
+        const tokenId = ethers.BigNumber.from(labelHash).toString();
+        const expires = new BigNumber(
+          String(await ens.functions.nameExpires(tokenId))
+        ).toNumber();
```
Comment (vgorkavenko) [inline ethereum-operations/src/agent-dao-ops.ts:443]: Why not just parse `Number` from `String`?

File: ethereum-operations/src/agent-dao-ops.ts (hunk 2 above)
Comment (dgusakov) [reply ethereum-operations/src/agent-dao-ops.ts:17]: Added

File: ethereum-operations/src/agent-dao-ops.ts (hunk 3 above)
Comment (dgusakov) [reply ethereum-operations/src/agent-dao-ops.ts:443]: Just a common approach

File: ethereum-financial/src/agent-pools-balances.ts (hunk 1 above)
Comment (dgusakov) [reply ethereum-financial/src/agent-pools-balances.ts:349]: The proposed code is not equal to the original one.

Comment (vgorkavenko) [reply ethereum-financial/src/agent-pools-balances.ts:349]: yep. fixed

File: ethereum-financial/src/agent-pools-balances.ts [hunk 4]
```diff
@@ -449,10 +462,13 @@ async function handleBalancerPoolSize(
...
```
Comment (vgorkavenko) [inline ethereum-financial/src/agent-pools-balances.ts:471]: the same here

Comment (dgusakov) [reply ethereum-financial/src/agent-pools-balances.ts]: Fixed

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**